src/styles/sass/main.scss

```diff
@@ -6,87 +6,319 @@
+      // }
+
+      &.contents {
+        top: -0.55rem;
+
+        @media (min-width: 30rem) {
+          display: inline-block;
+          position: relative;
+          top: -0.35rem;
+        }
+      }
+    }
+
+    // @media (min-width: 30rem) {
+    //   .setting-text {
+    //     display: inline-block;
+    //     position: relative;
+    //     top: -0.8rem;
+    //   }
+    // }
+
+    &.top {
+      background-color: $nypl-white;
+      border-bottom: 1px solid $nypl-blue-dark;
+
+      li:nth-child(1) {
+        display: none;
+      }
+
+      li:nth-child(3) {
+        float: right;
+      }
+    }
+    // Bottom Controls
+    // the, fixed bar and the arrows.
+    &.bottom {
+      background-color: darken($nypl-white, 2.5%);
+      border-top: 0.16rem solid $nypl-blue-dark;
+      bottom: 0;
+      display: block;
+      height: 3.2rem;
+      padding-top: 0.625rem;
+      position: fixed;
+      width: 100%;
+
+      @media (min-width: 30rem) {
+        padding-top: 0;
+        }
+
+      li {
+        border: 0;
+        float: none;
+        padding: 0;
+
+        @media (min-width: 30rem) {
+          &:nth-child(1) {
+            float: left;
+          }
+
+          &:nth-child(3) {
+            float: right;
+          }
+        }
+      }
+
+      // The Chapters label found only on mobile
+      li[aria-label="chapters"] {
+        margin: 0 20%;
+        position: relative;
+        top: -0.34rem;
+        text-align: center;
+
+        @media (min-width: 30rem) {
+          display: none;
+          }
+        }
+
+      @media (min-width: 30rem) {
+        //background: lighten($nypl-blue-dark, 50%);
+        }
+
+      .icon {
+        height: 1.5rem;
+        width: 1.5rem;
+
+        @media (min-width: 30rem) {
+          position: relative;
+          top: 0.375rem;
+        }
+      }
+
+      .chapter-control {
+        display: none;
+
+        @media (min-width: 30rem) {
+          display: inline-block;
+          }
+        }
+      }
     }
 
   ul li {
+    border: 0.0625rem solid $nypl-blue-dark;
+    border-radius: 0.3125rem;
+    display: inline-block;
     float: left;
+    padding: 0.2rem 0.65rem 0;
     margin: 0.625rem;
+
+    // @media (min-width: 30rem) {
+    //   border: 1px solid $nypl-white;
+    // }
   }
 
   a {
-    color: $nypl-white;
+    color: $nypl-blue-dark;
+    text-decoration: none;
     }
   }
 
 .loading {
+  background-color: darken($nypl-white, 2.5%);
+  border: 1px solid $nypl-blue;
+  border-radius: 0.3125rem;
+  color: $nypl-blue;
+  font-family: $sans-serif-stack;
+  height: 2.5rem;
+  left: calc(50% - 6.25rem);
+  padding-top: 1.25rem;
   position: fixed;
-  width: 12.5rem;
-  height: 12.5rem;
   text-align: center;
-  left: calc(50% - 6.25rem);
   top: calc(50% - 6.25rem);
-  background-color: $nypl-gray-cool;
+  width: 12.5rem;
 }
 
 .controls-view {
+  font-family: $sans-serif-stack;
   height: calc(100% - 2rem);
   left: 0;
   overflow: scroll;
   position: fixed;
```

Comment:
add this rule: 

`-webkit-overflow-scrolling:touch`

so that iOS shows the smooth-enabled scrolling… this is iOS only but still we should take advantage of it